### PR TITLE
Simon CMV (supersedes #48)

### PR DESCRIPTION
**Pull request by simonleandergrimm**
Supersedes #48 
 - Edited comments
 - Dropped active estimate of CMV. @jeffkaufman, feel free to spend 10 mins to look at #48 and the active estimate therein. I currently do not want to spend >30 min on evaluating and potentially improving upon this estimate, though happy to have my mind changed.

### CODE REVIEW

File: pathogens/CMV.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+in the body for life  after the acute phase, and may reactivate later under
+certain circumstances such as when the immune system is weakened.
+"""
+pathogen_chars = PathogenChars(
+    na_type=NAType.DNA,
+    enveloped=Enveloped.ENVELOPED,
+    taxid=TaxID(10359),
+)
+
+ger_adult_seroprevalence_estimate = Prevalence(
```
Comment (jeffkaufman) [inline pathogens/CMV.py:20]: Normally we abbreviate Germany as `de`, since that it's (3166-1 alpha 2) country code

File: pathogens/CMV.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    # Estimate is for 18-79 year olds. In reality, this implies slightly lower
+    # than 57% for the overall population including children
+    infections_per_100k=0.567 * 100000,
```
Comment (jeffkaufman) [inline pathogens/CMV.py:23]: 100_000

File: pathogens/CMV.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+)
+
+nhanes_6_to_49_US_seroprevalence = Prevalence(
+    infections_per_100k=0.504 * 100_000,
+    country="US",
+    active=Active.LATENT,
+    number_of_participants=15_310,
+    source="https://pubmed.ncbi.nlm.nih.gov/20426575/#:~:text=the%20overall%20age%2Dadjusted%20CMV%20seroprevalence%20was%2050.4%25",
+    start_date="1999",
+    end_date="2004",
```
Comment (jeffkaufman) [inline pathogens/CMV.py:55]: Is there no more recent NHANES data covering CMV?

**Review by jeffkaufman — CHANGES_REQUESTED** [review]

File: pathogens/CMV.py (hunk 3 above)
Comment (simonleandergrimm) [reply pathogens/CMV.py:55]: No there isn't, they stopped taking antibody measurements for all participants after 2004. The only available data is for [1-5yo in 2011/2012 ](https://wwwn.cdc.gov/Nchs/Nhanes/2011-2012/CMV_G.htm) and [2017-2020](https://wwwn.cdc.gov/Nchs/Nhanes/2017-2018/P_CMV.htm)

File: pathogens/cmv.py [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+adult_seroprevalence_raleigh_durham_US = Prevalence(
+    infections_per_100k=0.56 * 100_000,
+    # Overall CMV seroprevalence was 56.7% (95%CI: 54.8–58.7%), with a higher
+    # seroprevalence in women (62.3%) than in men (51.0%). Seroprevalence
+    # increased with age: from 31.8% to 63.7% in men and from 44.1% to 77.6%
+    #  in women when comparing the 18–29 with the 70–79 year age-group
+    number_of_participants=694,
+    date="2020",
+    country="US",
```
Comment (jeffkaufman) [inline pathogens/cmv.py:40]: `"United States"`

File: pathogens/cmv.py [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+)
+
+nhanes_6_to_49_US_urine_shedding = Prevalence(
+    infections_per_100k=0.0383 * 100_000,
+    # Among 6,828 CMV IgG-positive subjects tested, 537 had CMV DNA detected
+    # in urine—a shedding prevalence of 9.70%. Among persons 6–49 years,
+    # shedding prevalence was 3.83%. The prevalence of urinary shedding was
+    # inversely associated with increasing age (26.60%, 6.50%, and 3.45% in
+    # CMV IgG-positive subjects aged 6–11, 12–19, and 20–49 years, respectively
+    country="US",
```
Comment (jeffkaufman) [inline pathogens/cmv.py:65]: `"United States"`

File: pathogens/cmv.py [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+    state="North Carolina",
+    # Study is actually from the Raleigh-Durham-Chapel Hill Metropolitan Area
+    county="Raleigh County",
+    active=Active.LATENT,
+    source="https://www.frontiersin.org/articles/10.3389/fcimb.2020.00334/full#:~:text=Methods%3A%20We,population%20was%2056%25",
+)
+
+nhanes_6_to_49_US_seroprevalence = Prevalence(
+    infections_per_100k=0.504 * 100_000,
+    country="US",
```
Comment (jeffkaufman) [inline pathogens/cmv.py:50]: `"United States"`

**Review by jeffkaufman — APPROVED** [review]

**Pull request merged**